src/main/java/duke/util/Parser.java

```diff
@@ -51,6 +53,12 @@ public Command parseInput(String rawInput) throws DukeException {
             } catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
                 throw new DukeException("Please use 'delete i' to delete the i-th task in the list.");
             }
+        case "find":
+            if (!rawInput.contains(" ")) {
```

Comment:
I did not consider the case where the search term is empty.
Good idea, I will try it.